### ISSUE TITLE
KEY_ALT isn't detected

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot#12073]
Godot 2.1.3, 2.1.4
Windows 10 64 bits

The following code doesn't print if I press Alt:
```gdscript
func _ready():
	set_process(true)

func _process(delta):
	if Input.is_key_pressed(KEY_ALT):
		print("TEST")
```

It works with another key like KEY_CONTROL.
I didnt test yet on 3.0,

**Comment by Noshyaar** [comment]
Works fine in 3.0 latest master for both ALT keys

**Comment by Ranoller** [comment]
Key alt have "bad binded" value (sorry for the english, i can´t find best way to say that). Actual constant value (in 2.1) for alt key is 16777262.... try replace KEY_ALT with that.

**Comment by ISylvox** [comment]
I guess Key_ALT KeyCode is a windows specific issue which happened on all version of Godot (1.x ; 2.x ; 3.0) as it's wrongly paired with its Virtual-KeyCode (never changed since Godot opensourced). I had fixed this issue two weeks ago in my local repo, but didn't submit a PR (was waiting for someone made an issue about this).

**Comment by ISylvox** [comment]
This Key_ALT issue is also effecting 3.0 on Windows. And as for my test, already solved by my merged PR(both for 2.1 and 3.0). I think this issue is now safe to be closed(no?). :-)

**Comment by ISylvox** [comment]
This Key_ALT issue is also effecting 3.0 on Windows. And as for my test, already solved by my merged PR(both for 2.1 and 3.0). I think this issue is now safe to be closed(no?). :-)

Kindly CC @Zylann

**Comment by ISylvox** [comment]
This Key_ALT issue is also effecting 3.0 on Windows. And as for my test, already solved by my merged PR(both for 2.1 and 3.0). I think this issue is now safe to be closed(no?). :-)

pretty ping @Zylann @Noshyaar
